docs/blog/_posts/2018-11-27-11th-dotty-milestone-release.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.11.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-11-17
+---
+
+Today we are excited to release Dotty version 0.11.0-RC1.
+This release serves as a technology preview that demonstrates new language features and the
+compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs well together,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 11th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+## What’s new in the 0.11.0-RC1 technology preview?
+
+### Opaque Type Aliases
+
+Opaque types aliases provide type abstraction without any overhead. Example:
+
+```scala
+opaque type Logarithm = Double
+```
+
+This introduces `Logarithm` as a new type, which is implemented as `Double` but is different from
+it. The fact that `Logarithm` is the same as `Double` is only known in the companion object of
+`Logarithm`. Here is a possible companion object:
+
+```scala
+object Logarithm {
+
+  // These are the ways to lift to the logarithm type
+  def apply(d: Double): Logarithm = math.log(d)
+
+  def safe(d: Double): Option[Logarithm] =
+    if (d > 0.0) Some(math.log(d)) else None
+
+  // This is the first way to unlift the logarithm type
+  def exponent(l: Logarithm): Double = l
+
+  // Extension methods define opaque types' public APIs
+  implicit class LogarithmOps(val `this`: Logarithm) extends AnyVal {
+    // This is the second way to unlift the logarithm type
+    def toDouble: Double = math.exp(`this`)
+    def +(that: Logarithm): Logarithm = Logarithm(math.exp(`this`) + math.exp(that))
+    def *(that: Logarithm): Logarithm = Logarithm(`this` + that)
```

Comment:
I might be mistaken, but shouldn't this be something like this (to counter the `math.log` from `Logarithm.apply`)?
```suggestion
    def *(that: Logarithm): Logarithm = Logarithm(math.exp(`this` + that))
```

Reply:
@nightscape Thanks for the suggestion. I have replaced the `Logarithm` example with a simpler one